src/Agent.Sdk/SecretMasking/OssSecretMasker.cs

```diff
@@ -1,10 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
+using Microsoft.Security.Utilities;
```

Comment:
>Utilities [](http://example.com/codeflow?start=25&length=9)

SEV3 : i think other code in this repo prefers System.* directives appear before MS